.github/ISSUE_TEMPLATE/release_new_version.md

```diff
@@ -0,0 +1,69 @@
+- [ ] Finish App translations
+  - [ ] Review PRs <details><summary>Checklist for the PRs</summary> <!-- FIXME: This should be moved to the admin wiki and linked here -->
+     ```
+     - [ ] Translator listed in the `src/util.cpp` **optionally add link to PR or code**
+     - [ ] Punctuation and spacing consistent
+     - [ ] Signal words consistent ("ASIO", "Buffer")
+     - [ ] App translations: No untranslated strings (`grep unfinished -5 src/res/translation/translation_$TRANSLATION*.ts`)
+     - [ ] App translations: Only a single `.ts` file checked in (`.qm` in addition is also OK)
+     - [ ] Installer translations: Passes `tools/check-wininstaller-translations.sh`
+     ```
+     </details>
+  - [ ] Wait for all PRs to be merged. If translations are missing after the targeted translation completion date, this is ok. They can be updated with the next release again.
+  - [ ] Check for conflicting accelerator keys (see `tools/checkkeys.pl`)
+  - [ ] Generate `.qm` files via `lrelease Jamulus.pro`
+- [ ] [Tag a release candidate](https://github.com/jamulussoftware/jamulus/blob/master/RELEASE-PROCESS.md#steps-for-a-specific-release)
+- [ ] Announce the release candidate on Github Discussions. Pin the thread. Unpin and lock the beta thread.
+- [ ] Draft an announcement, include all contributors via `tools/get_release_contributors.py`
+- [ ] [Update the version number in `Jamulus.pro` and add the release date to the Changelog header and commit](https://github.com/jamulussoftware/jamulus/blob/master/RELEASE-PROCESS.md#steps-for-a-specific-release)
+- [ ] Tag this commit as `r3_y_z`
+- [ ] Wait for the build to complete
+- [ ] Do a smoke test for Windows/Mac/Linux -- Do the binaries start/connect properly? Can earlier Jamulus versions properly connect to a server based on the new release?
+- [ ] [Force tag that tag as `latest` and push.](https://github.com/jamulussoftware/jamulus/blob/master/RELEASE-PROCESS.md#if-this-is-a-proper-release-move-the-latest-tag)
+- [ ] Upload the artifacts to SourceForge and link latest
+- [ ] [Prepare `Jamulus.pro` (`dev` suffix) and ChangeLog (add a header) for the next release](https://github.com/jamulussoftware/jamulus/blob/master/RELEASE-PROCESS.md#make-the-master-branch-ready-for-post-release-development)
+- [ ] Trigger the update notification by updating the both Update Check Servers with the new version
+- [ ] Announce the new release with a summary of changes (+ link to the changelog for details) and a link to the download page
+  - [ ] On Github Discussions in the Announcements section. Lock the announcement thread. Pin the thread. Unpin and lock release candidate thread.
+  - [ ] On Facebook in the group "Jamulus (official group)". Turn off replies.
+- [ ] Update download links on the website by editing `config.yml`
+- [ ] Publish Website release by merging `translate3_y_z` into `changes`
```

Comment:
```suggestion
- [ ] Publish Website release by merging `translate3_y_z` into `release`
```

Reply:
Thanks, I meant to write that but forgot that update. :/
Done now.